src/gui/mod.rs

```diff
@@ -350,6 +352,10 @@ impl Default for Ajour {
             pending_confirmation: None,
             zstd_compression_level_slider_state: Default::default(),
             share_state: Default::default(),
+            onboarding_status_button_state: Default::default(),
+            addons_status_button_state: Default::default(),
+            weakauras_status_button_state: Default::default(),
+            catalog_status_button_state: Default::default(),
```

Comment:
Does this have to be 4 buttons or can it be one? Since it's only passed once / on different pattern branches.

Reply:
Good point. Fixed.